dev/init_db.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+su -u postgres
```

Comment:
At what point and who runs this script? 
I thought postgres docker container should do it by itself

Reply:
I have re-read about it and actually you are right. I have removed the script.